research/object_detection/model_lib.py

```diff
@@ -695,6 +705,12 @@ def create_train_and_eval_specs(train_input_fn,
       exporter_name = '{}_{}'.format(final_exporter_name, eval_spec_name)
     exporter = tf.estimator.FinalExporter(
         name=exporter_name, serving_input_receiver_fn=predict_input_fn)
+    if export_best_model:
+      exporter = tf.estimator.BestExporter(name="best_exporter",
+                                           serving_input_receiver_fn=predict_input_fn,
+                                           event_file_pattern="eval_0/*.tfevents.*",
```

Comment:
event_file_pattern is hardcoded to have "eval_0" prefix, is that guaranteed to select the right eval events?